view/src/test/java/org/xcolab/entity/utils/LinkUtilsTest.java

```diff
@@ -0,0 +1,46 @@
+package org.xcolab.entity.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class LinkUtilsTest {
```

Comment:
Please add a test for a blank URL string, e.g. `" "`.

Reply:
I rewrote the function to now parse and validate all input URIs, this way this test makes more sense (as otherwise we'd just be double checking the pass-through). I think this makes the method more straight-forward: it will either return a relative URI or throw an `IllegalArgumentException`.